src/components/taskDetails/taskDependency/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ChangeEvent, FC, useState } from 'react';
+import { Textarea } from '..';
+import { TaskDependencyProps } from '@/interfaces/taskDetails.type';
+import DependencyList from '@/components/taskDetails/taskDependency/DependencyList';
+
+const TaskDependency: FC<TaskDependencyProps> = ({
+    taskDependencyIds,
+    isEditing,
+    updatedDependencies,
+    handleChange,
+}) => {
+    const [editedDependencies, setEditedDependencies] =
+        useState<string[]>(updatedDependencies);
+
+    const handleDependenciesChange = (
+        event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+    ) => {
+        const { value } = event.target;
+        const updatedDependencies = value
+            .split(',')
+            .map((taskId) => taskId.trim());
```

Comment:
This logic is repeated elsewhere as well. Please move it to utils

Reply:
where it is repeated